test_httpbin_ip.py

```diff
@@ -48,3 +48,10 @@ def test_basic_auth_2(base_url, user, password):
     r = get(base_url + 'basic-auth/{0}/{1}'.format(user, password), auth=(user, password))
     assert r.status_code == 200
     assert r.json() == {'authenticated': True, 'user': user}
+
+
+def test_deflated(base_url):
+    r = get(base_url + 'deflate')
+    assert r.json()['method'] == 'GET'
+    assert r.status_code == 200
+    assert r.json()['headers']['Host'] == base_url[7:-1]
```

Comment:
not bad!
can you expand the test and add few more checks?
there is quiet a bit of data there:
```json
{
  "deflated": true, 
  "headers": {
    "Accept": "*/*", 
    "Accept-Encoding": "gzip, deflate", 
    "Connection": "close", 
    "Host": "httpbin.org", 
    "User-Agent": "python-requests/2.18.4"
  }, 
  "method": "GET", 
  "origin": "55.134.189.123"
}
```

additionally you can compare response body headers with actual headers (r.json()['headers'] with r.headers)

Reply:
and of course try other http methods